Source/Extensions/Blazorise.DataGrid/Enums/RowOverlayPosition.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Blazorise.DataGrid.Enums
+{
+    /// <summary>
+    /// Defines the position of the row overlay
+    /// </summary>
+    public enum RowOverlayPosition
+    {        
+        /// <summary>
+        /// Overlay start at left of row.
+        /// </summary>
+
+        Left,
```

Comment:
Prefer naming `Start` and `End` instead of `Left` and `Right`.

Reply:
Renamed as requested

Reply:
You're quick :)